llama-index-core/llama_index/core/base/llms/types.py

```diff
@@ -44,21 +30,15 @@ class MessageRole(str, Enum):
     MODEL = "model"
 
 
-# ===== Generic Model Input - Chat =====
-class ContentBlockTypes(str, Enum):
-    TEXT = "text"
-    IMAGE = "image"
```

Comment:
fyi, the type was mainly to give a solid discriminator for pydantic when serializing and deserializing chat messages/blocks

As long as two blocks don't have the exact same fields, I think this should be safe though

Reply:
That's not expected indeed, audio will have its own set of fields.

Reply:
I'm using a custom field serializer to ensure we discriminate the actual block type

Reply:
Had troubles with deserialization, bringing back a discriminator field